src/components/AddBook.jsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import { v4 as uuidv4 } from 'uuid';
-import PropTypes from 'prop-types';
 import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addNewBook } from '../redux/books/bookSlice';
+
+function AddBook() {
```

Comment:
- [ ] Kindly look through your app. I cannot add a book. Regardless of what I submit, the book always defaults to Hunger Games. Kindly ensure the added books are reflected on the page.